### ISSUE TITLE
Convert table in Chapter 7 from an image to an actual table

### ISSUE THREAD

**Issue by burkemlou** [ELIXIR-FAIR-Training/FAIR-training-handbook#30]
The table of metadata in Chapter 7 is currently an image. This should be converted to a table to make it accessible for screen readers.

**Comment by burkemlou** [comment]
Done. Table has been converted to markdown.
